src/script/script.cpp

```diff
@@ -5,18 +5,11 @@
 
 #include "script.h"
 
+#include "script/interpreter.h"
```

Comment:
Yes, script should probably not depend on the interpreter.
I think we should move the disassembly logic out of the core/consensus data structure.